src/Hooks/ResizeWindow.js

```diff
@@ -0,0 +1,20 @@
+import { useState, useEffect } from 'react';
+
+export function useResizeWindow(setMaxEntriesPerPage, setCurrentPage) {
```

Comment:
One more step, let's say another component wanted to use this logic to check the window width and do something when it changes to mobile and back - what if they don't use `maxEntriesPerPage` and `currentPage`?
Let's make this function a bit more generic and use the result to do what we want in other components